code/modules/projectiles/ammo_datums.dm

```diff
@@ -1810,9 +1810,9 @@ datum/ammo/bullet/revolver/tp44
 /datum/ammo/energy/lasgun/marine/sniper
 	name = "sniper laser bolt"
 	hud_state = "laser_sniper"
-	damage = 60
+	damage = 85
```

Comment:
this seems completly uneeded

Reply:
I wanted to give it a damage boost because it now loses damage for each marine it passes through, however you're right that +25 damage is unneeded. How about 70 instead? If that is still too much I'll just leave it at 60.